src/js/include/queue-manager.js

```diff
@@ -0,0 +1,84 @@
+import { updateResultFailure, updateResult } from "./render";
+import $ from "jquery";
+
+export function initQueue(itemsToScan, activeOnly) {
+  // Reset the queue.
+  window.phpcompat.queue = [];
+
+  itemsToScan.plugins.forEach((plugin) => {
+    if ("yes" === plugin.active || "no" === activeOnly) {
+      window.phpcompat.queue.push({
+        ...plugin,
+        type: "plugin",
+      });
+    }
+  });
+
+  itemsToScan.themes.forEach((theme) => {
+    if ("yes" === theme.active || "no" === activeOnly) {
+      window.phpcompat.queue.push({
+        ...theme,
+        type: "theme",
+      });
+    }
+  });
+}
+
+export function executeJob(job, cb) {
+  // TODO indicate current plugin/theme in progress
+
+  var endpoint = `https://staging.wptide.org/api/v1/audit/wporg/${job.type}/${job.slug}/${job.version}?reports=phpcs_phpcompatibilitywp`;
```

Comment:
I'm assuming this endpoint will be updated to a non-staging version at some point. May be worth adding a `TODO` statement here to help us not forget

Reply:
Thanks, I've added TODO for that